module/VuFind/src/VuFind/Search/EDS/Options.php

```diff
@@ -320,104 +465,59 @@ protected function setCommonSettings($searchSettings, $setting, $list, $target)
     }
 
     /**
-     * Load options from the configuration file. These will override the defaults set
-     * from the values in the Info method. (If the values set in the config files in
-     * not a 'valid' EDS API value, it will be ignored.
-     *
-     * @param \Laminas\Config\Config $searchSettings Configuration
+     * Load options from the configuration file.
      *
      * @return void
      */
-    protected function setOptionsFromConfig($searchSettings)
+    protected function setOptionsFromConfig()
     {
-        if (isset($searchSettings->General->default_limit)) {
-            $this->defaultLimit = $searchSettings->General->default_limit;
+        if (isset($this->searchSettings->General->default_limit)) {
+            $this->defaultLimit = $this->searchSettings->General->default_limit;
         }
-        if (isset($searchSettings->General->limit_options)) {
+        if (isset($this->searchSettings->General->limit_options)) {
             $this->limitOptions
-                = explode(",", $searchSettings->General->limit_options);
+                = explode(",", $this->searchSettings->General->limit_options);
         }
 
         // Set up highlighting preference
-        if (isset($searchSettings->General->highlighting)) {
-            $this->highlight = $searchSettings->General->highlighting;
+        if (isset($this->searchSettings->General->highlighting)) {
+            $this->highlight = $this->searchSettings->General->highlighting;
         }
 
         // Set up facet preferences
-        if (isset($searchSettings->General->include_facets)) {
-            $this->includeFacets = $searchSettings->General->include_facets;
+        if (isset($this->searchSettings->General->include_facets)) {
+            $this->includeFacets = $this->searchSettings->General->include_facets;
```

Comment:
Unless I'm overlooking something, this seems like an orphaned property/config that never gets used for anything. Would you agree?

Reply:
You're right, cleaned up. The backend's SearchRequestModel is still supports it just in case.